locales/en/messages.json

```diff
@@ -4568,14 +4559,26 @@
     "failsafeChannelFallbackSettingsHold": {
         "message": "<strong>Hold</strong> means maintain the last good value received. <strong>Set</strong> means the value given here will be used"
     },
+    "failsafeChannelFallbackSettingsValueAuto": {
+        "message": "Auto",
+        "description": "Text for Auto option"
+    },
+    "failsafeChannelFallbackSettingsValueHold": {
+        "message": "Hold",
+        "description": "Text for Hold option"
+    },
+    "failsafeChannelFallbackSettingsValueSet": {
+        "message": "Set",
+        "description": "Text for Set option"
+    },
     "failsafeStageTwoSettingsTitle": {
         "message": "Stage 2 - Settings"
     },
     "failsafeDelayItem": {
         "message": "Guard time for stage 2 activation after signal lost [seconds]"
     },
     "failsafeDelayHelp": {
-        "message": "Time for stage 1 to wait for recovery"
+        "message": "Time for stage 2 to wait for recovery"
```

Comment:
This should be stage 1.

Reply:
Sure @ctzsnooze would have noticed

Reply:
Waiting for comment from @ctzsnooze or just rollback ? :-)

Reply:
Good catch, the wording is unclear.
What actually happens is that a signal loss more than than 30ms is starts a counter, and after 100ms of no signal we initiate Stage 1 behaviour.  We stay in Stage 1 until the failsafe delay time expires, and then enter Stage 2.  While in Stage 1, signal recovery causes immediate exit from Stage 1 behaviour.

Perhaps we can say, "The period of time in Stage 1 failsafe after signal loss.  If the delay time expires without return of signal, Stage 2 failsafe will start."

Does that sound OK?

Reply:
Good suggestion, now:
![image](https://github.com/betaflight/betaflight-configurator/assets/99370924/841cda3e-5ffa-427f-9c35-eda9b8f763c2)